service-api/Dockerfile

```diff
@@ -1,7 +1,58 @@
 FROM php:8.2-fpm-alpine AS app
 
-RUN mkdir /app
-WORKDIR /app
-COPY index.php .
+RUN apk --no-cache add postgresql-dev fcgi icu-dev autoconf linux-headers $PHPIZE_DEPS \
+  && docker-php-ext-configure intl \
+  && docker-php-ext-install pdo pdo_pgsql opcache intl sockets zip \
```

Comment:
You probably won't need all of these, since intent is to use dynamo for storage and we won't be doing any zip work in this app. I'd remove all but opcache until you need them.